config/rbac/role.yaml

```diff
@@ -27,6 +27,18 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Side question related to the whole file.

This file is fully generated and any modification will be overwritten by kubebuilder.

Should we consider this file as a build artifact ? Is it part of good practice to still track it in the repository ?

Reply:
I don't have a strong opinion.. one advantage to keep it in git is to have a quick & easy way to see which permissions are required, without having to fetch repo and deploy. But that's all, if people think it's better to remove (e.g. to have less PR conflicts) i'm fine either way